packages/core-utils/package.json

```diff
@@ -13,11 +13,13 @@
     "@opentripplanner/geocoder": "^1.0.2",
     "@turf/along": "^6.0.1",
     "bowser": "^2.7.0",
+    "date-fns": "^2.23.0",
+    "date-fns-tz": "^1.1.4",
```

Comment:
According to the docs, `date-fns-tz` uses the [Intl](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl) API. This isn't fully supported by IE/Edge and might need a polyfill depending on what the lowest version of IE/Edge is that we need to support. Also, the GitHub page for this recommends using Node 13+ with this for full compatibility. I'm somewhat inclined to call this a breaking change if implementing this. And if so, we should bump to at least node 13 in the `engines` section of this file.

Reply:
I looked at the `date-fns-tz` code and it seems like the only part of the Intl API that is used is the [Intl.DateTimeFormat()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat) constructor which according to [caniuse](https://caniuse.com/?search=Intl.DateTimeFormat) currently has 95.89% global support and supports nearly any browser released after the year 2017.

Reply:
Note has been added to the readme and package.json engines are updated in 55e76870